fineract-provider/src/main/java/org/apache/fineract/useradministration/service/AppUserReadPlatformServiceImpl.java

```diff
@@ -55,7 +52,6 @@ public class AppUserReadPlatformServiceImpl implements AppUserReadPlatformServic
     private final AppUserRepository appUserRepository;
     private final StaffReadPlatformService staffReadPlatformService;
 
-    @Autowired
     public AppUserReadPlatformServiceImpl(final PlatformSecurityContext context, final JdbcTemplate jdbcTemplate,
```

Comment:
Just a minor thing: let's take the opportunity to clean up situations like these... not necessarily wrong, but we can use the Lombok @RequiredArgsConstructor class annotation and get rid of the constructor boilerplate code. Saves us again a bunch of unnecessary lines of code.

Reply:
Done.